core/src/banking_trace.rs

```diff
@@ -59,6 +59,11 @@ pub struct BankingTracer {
     active_tracer: Option<ActiveTracer>,
 }
 
+#[cfg_attr(
+    feature = "frozen-abi",
+    derive(AbiExample),
+    frozen_abi(digest = "DAdZnX6ijBWaxKAyksq4nJa6PAZqT4RShZqLWTtNvyAM")
+)]
```

Comment:
this was temporarily removed by #3974. but it seems the reported build issue disappeared..